geomesa-spark/geomesa-spark-sql/src/test/scala/org/locationtech/geomesa/spark/SparkSQLGeometryConstructorsTest.scala

```diff
@@ -53,14 +55,72 @@ class SparkSQLGeometryConstructorsTest extends Specification with LazyLogging {
       df.collect().length mustEqual 3
     }
 
+    "st_box2DFromGeoHash" >> {
+      val r = sc.sql(
+        s"""
+           |select st_box2DFromGeoHash('u2sux', 25)
```

Comment:
Perhaps we should consider adding an example that can be verified independently; such as the "ezs42" example  described on the geohash wikipedia page:  https://en.wikipedia.org/wiki/Geohash